README.md

```diff
@@ -36,3 +36,6 @@ when creating a pull request, you need to inform us of what you have done or cha
 # Keeping your Fork in Sync
 
 As more content is added to this repository and more people contribute, your fork will fall out of date. This can sometimes lead to tricky situations with your pull requests. It's recommended to occasionally sync your fork with this [repository](https://help.github.com/articles/syncing-a-fork/).
+
```

Comment:
Welldone so far @kolawolejohn, can you remove these 3 spaces still showing here. The README.md file shouldn't show in your PR at all.

Reply:
I have done that. Thanks

Reply:
> I have done that. Thanks
Okay, I can still see the ReadMe in the files changed session though, can you try to make sure it doesn't show at all?